package.json

```diff
@@ -14,6 +14,8 @@
     "prepare": "husky install",
     "precommit": "run-p 'format:staged'",
     "watch:common": "wsrun --stages --exclude-missing watch",
+    "watch:api": "run-p 'watch:common' 'start:api:local'",
```

Comment:
I thought we'd rely on the development tools provided by next/nest rather than using watch, I'm not familiar with a setup like this does it work similarly?

Reply:
this is ugly implementation ..... not optimised.